pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/security/ApexDangerousMethodsRule.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.apex.rule.security;
+
+import java.util.List;
+
+import net.sourceforge.pmd.lang.apex.ast.ASTMethodCallExpression;
+import net.sourceforge.pmd.lang.apex.ast.ASTUserClass;
+import net.sourceforge.pmd.lang.apex.rule.AbstractApexRule;
+
+/**
+ * Flags dangerous method calls, e.g. FinancialForce
+ * Configuration.disableTriggerCRUDSecurity()
+ * 
+ * 
+ * @author sergey.gorbaty
+ *
+ */
+public class ApexDangerousMethodsRule extends AbstractApexRule {
+	private static final String DISABLE_CRUD = "disableTriggerCRUDSecurity";
+	private static final String CONFIGURATION = "Configuration";
+
+	public ApexDangerousMethodsRule() {
+		setProperty(CODECLIMATE_CATEGORIES, new String[] { "Security" });
+		setProperty(CODECLIMATE_REMEDIATION_MULTIPLIER, 100);
+		setProperty(CODECLIMATE_BLOCK_HIGHLIGHTING, false);
+
```

Comment:
@jsotuyod w.r.t. the actual method call, Apex does not have imports per se, once a package is installed into an organization, if a class is declared as global, it could be called from other classes. Also, there's pretty much never a good reason to call this method. When it's called, it's turning off all global level checks AND requires the developer to provide all check locally, which they might not do when they rely on FinancialForce API.
It's good to make someone aware they're calling something dangerous, like Java unsafe.